zebrad/src/components/rpc/config.rs

```diff
@@ -0,0 +1,27 @@
+//! User-configurable RPC parameters.
+
+use std::net::SocketAddr;
+
+use serde::{Deserialize, Serialize};
+
+/// RPC configuration section.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields, default)]
+pub struct Config {
+    /// Turn on/off the RPC server
+    pub listen: bool,
+
+    /// IP address and port for the RPC server
+    pub listen_addr: SocketAddr,
```

Comment:
Yes, i think this will make it with only one option so i added it at https://github.com/ZcashFoundation/zebra/pull/3589/commits/27ca7bab8d23f61d1436460e92de793eacb2cd6c

However, by now the default option for the RPC server is ON. I think this can be changed if we add a note that the standard port for this is `8232` for the mainnet.